elastic/kibana
# [Response Ops][Alerting] Adding evaluation threshold to alert payload for ES query rule

Resolves https://github.com/elastic/kibana/issues/166986

## Summary

Adding `kibana.alert.evalution.threshold` to the alert payload for the ES query rule. This is the field that's shown in the alert details view in Observability. To show this, we add `ALERT_EVALUATION_CONDITIONS` to the stack alerts mapping, using the same mapping type as the observability rule types. This is typed as a `scaled_float` which is expecting a single value, so the threshold is set in the alert payload only when the threshold is a single value. I will open a followup issue for handling multi-valued thresholds. https://github.com/elastic/kibana/issues/172714

<img width="1064" alt="Screenshot 2023-11-20 at 1 10 05 PM" src="https://github.com/elastic/kibana/assets/13104637/e265a9e8-4bbf-4d3e-a6bc-e69b774c7574">


## To Verify

Create an ES query rule with a single threshold that triggers an alert and give it a Metrics or Logs visibility. Let it run and then look at the alert details for the alert from the Observability alert table. The `Expected Value` row should be populated.
